ublox/tests/parser_tests.rs

```diff
@@ -195,6 +195,63 @@ fn test_parse_cfg_nav5() {
     assert!(found);
 }
 
+#[test]
+#[cfg(feature = "serde")]
+fn test_esf_meas_serialize() {
+    let ret = [
+        181, 98, 16, 2, 16, 0, 243, 121, 129, 1, 24, 8, 0, 0, 77, 100, 0, 11, 211, 148, 129, 1,
```

Comment:
may be better if we have a builder for this instead of explicitly defining bytes but think the current coverage covers converting bytes already